langserver/langserver_test.go

```diff
@@ -1038,16 +1038,21 @@ func (h *Hello) Bye() int {
 }
 
 func TestServer(t *testing.T) {
-	GocodeCompletionEnabled = true
-
 	for label, test := range serverTestCases {
 		t.Run(label, func(t *testing.T) {
 			if test.skip {
 				t.Skip()
 				return
 			}
 
-			h := &LangHandler{HandlerShared: &HandlerShared{}}
+			cfg := NewDefaultConfig()
+			cfg.FuncSnippetEnabled = true
```

Comment:
This wasn't `true` previously. But the build is green so I assume it does no harm right?

Reply:
Good catch.  Since `FuncSnippetEnabled` is defaulted to `true` in `main.go`, I wonder if we should keep it `true` here also, since `GocodeCompletionEnabled` is `true` in this test.

It might make little difference in practice, but it does make the two worlds match a little more.

Any preference or another suggestion?

Reply:
Yeah leave it true.